spritz/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.novoda.spritz.sample">
+
+    <application
+        android:allowBackup="true"
```

Comment:
`allowBackup` should be `false` for a library demo app with nothing to back up

Reply:
Done in bec8bf8a5da89d03bf4646289c0d4ce3fe9464e4.